backend/box/box.go

```diff
@@ -402,24 +417,27 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 		CanHaveEmptyDirectories: true,
 	}).Fill(f)
 	f.srv.SetErrorHandler(errorHandler)
+	f.srv.SetHeader("Authorization", "Bearer "+f.opt.AccessToken)
```

Comment:
Do we want to do this always? It seems like we'd only want to set this if `f.opt.AccessToken` was valid

Reply:
It still worked even when using another auth method and having an empty AccessToken, but you're right, I probably shouldn't assume that'll always be the case